packages/client/src/v2-events/components/forms/FormFieldGenerator/FormSectionComponent.tsx

```diff
@@ -387,7 +387,7 @@ export function FormSectionComponent({
   const hasAnyValidationErrors = fieldsWithFormikSeparator.some((field) => {
     const fieldErrors = errors[field.id]?.errors
     const hasErrors = fieldErrors && fieldErrors.length > 0
-    return isFieldVisible(field, completeForm) && hasErrors
+    return isFieldVisible(field, completeForm, { locations: [] }) && hasErrors
```

Comment:
These I think need to pass in the actual populated validation context so `{ locations }` with the full list